src/Runner.Worker/Handlers/ScriptHandler.cs

```diff
@@ -237,6 +239,11 @@ public async Task RunAsync(ActionRunStage stage)
                 ExecutionContext.StepTelemetry.Action = shellCommand;
             }
 
+            if (ExecutionContext.JobContext.Container != null)
+            {
+                ExecutionContext.StepTelemetry.ContainerHookData = HostContext.GetService<IContainerHookManager>().GetContainerHookData();
```

Comment:
should we do this in a single place inside the ExecutioContext? so each caller doesn't have to do this by themselves.